README.md

```diff
@@ -1 +1,27 @@
-# pkg-node-logger
+# Spreaker Node Logger
+
+Pino-based Logger class that we use in all of Spraeker's nodeJS projects.
+
+
+## Install package
+
+`npm install @spreaker/node-logger`
```

Comment:
Maybe we can just call it `@spreaker/logger` being npm a node-only repo?

Reply:
(it's a common practice in npm to use the `node-` prefix for repos but then omitting it for npm packages)

Reply:
Fixed